habitat-baselines/habitat_baselines/config/default_structured_configs.py

```diff
@@ -345,6 +380,8 @@ class HabitatBaselinesConfig(HabitatBaselinesBaseConfig):
     # )
     # cmd_trailing_opts: List[str] = field(default_factory=list)
     trainer_name: str = "ppo"
+    updater_name: str = "PPO"
+    distrib_updater_name: str = "DDPPO"
```

Comment:
No, I think they are best here because they are needed to instantiate the updater. But this was a mistake, `rl_hierarchical` was supposed to change these properties.

Reply:
Can't these just be inferred from other configurations ?